apps/api/src/pkg/errors/openapi_responses.ts

```diff
@@ -46,6 +46,17 @@ export const openApiErrorResponses = {
       },
     },
   },
+  412: {
+    description:
+      "This response is sent when the request does not meet one or more preconditions required for fulfillment.",
```

Comment:
Something a bit detailed might be good. 

Example:
```sh
 description:  "The requested operation cannot be completed because certain conditions were not met. This typically occurs when a required resource state or version check fails."
```